pkg/testing/match/match.go

```diff
@@ -17,12 +17,22 @@ package match
 
 import (
 	"encoding/json"
+	"os"
 	"reflect"
 	"regexp"
 	"strings"
 	"testing"
 
 	"github.com/stretchr/testify/require"
+
+	eventtypes "github.com/inspektor-gadget/inspektor-gadget/pkg/types"
+)
+
+var DefaultTestComponent string
```

Comment:
This shouldn't be part of match. I'd recommend creating a new "component" pkg under testing that has this constant.